### ISSUE TITLE
Context test column name is None

### ISSUE THREAD

**Issue by hilcik** [fal-ai/fal#791]
**Describe the bug**
When accessing `context.current_model.tests` the column name is None if the tests are defined on table level instead of column level. For example these two tests below are doing the same thing, but only the one in `columns` key will have the column name metadata populated in `context`.

```
tests:
  - unique:
      column_name: col
columns:
  - name: col
    tests:
      - unique
```

**Your environment**
- OS: macOS Ventura 13.2.1
- Paste the following commands output:
```
fal --version
fal 0.8.2
dbt --version
Core:
  - installed: 1.4.5

```
- Adapter being used: athena: 1.4.1

**How to reproduce**
1. Define a test in a yml file for an existing dbt model on a table level and specify a column. For example:
```
models:
  - name: model
    tests:
      - unique:
          column_name: col
```
2.  run a python script accessing context.model.tests.column and it should be None even though in the yml file it's defined for column `col`.

**Expected behavior**
I would expect to see `column='col'`

**Actual behavior**
`column=None`

**Screenshots**

**Additional context**

**Comment by mederka** [comment]
Thanks for raising this, @hilcik, I'm looking into this.

**Comment by mederka** [comment]
Hi @hilcik :wave: , we just released fal==0.8.3, it should fix this issue. Please, let us know if you run into other problems.
